src/pages/ReimbursementAccount/BankAccountPlaidStep.js

```diff
@@ -51,6 +53,14 @@ function BankAccountPlaidStep(props) {
         return errorFields;
     }, []);
 
+    useEffect(() => {
```

Comment:
I'm not entirely sure how this works.

So, any time the Plaid Setup element is focused, or any time the `plaidData` gets updated we will run the `useEffect` method, correct?

What about when the element is unfocused (like on a blur)?